src/helperFunctions/web_interface.py

```diff
@@ -4,6 +4,7 @@
 from datetime import timedelta
 
 from common_helper_files import get_binary_from_file
+from matplotlib import cm, colors
```

Comment:
We have a matplotlib dependency elsewhere, right? Otherwise this needs an addition to the frontend install script (while keeping in mind that using matplotlib for only this color map would be a rather big overhead).